app/controllers/nqt_job_alerts_controller.rb

```diff
@@ -0,0 +1,31 @@
+class NqtJobAlertsController < ApplicationController
+  include ParameterSanitiser
+
+  def new
+    @nqt_job_alerts_form = NqtJobAlertsForm.new(nqt_job_alerts_params)
+  end
+
+  def create
+    @nqt_job_alerts_form = NqtJobAlertsForm.new(nqt_job_alerts_params)
+    subscription = Subscription.new(@nqt_job_alerts_form.job_alert_params)
+    @subscription = SubscriptionPresenter.new(subscription)
+
+    recaptcha_valid = verify_recaptcha(model: subscription, action: 'subscription')
+    subscription.recaptcha_score = recaptcha_reply['score'] if recaptcha_valid && recaptcha_reply
+
+    if @nqt_job_alerts_form.valid?
+      subscription.save
+      AuditSubscriptionCreationJob.perform_later(@subscription.to_row)
+      SubscriptionMailer.confirmation(subscription.id).deliver_later
+      return render :confirm
+    end
+
+    render :new
```

Comment:
To be more in line with rails I would put this in an `else` branch of the above condition and remove the return from
`return render :confirm`

Reply:
The Python in me is fighting to come out every day 😂